### PR TITLE
Add attribute filter_odt_journeys in api instance

### PR DESCRIPTION
**Pull request by kadhikari**
Add attribute filter_odt_journeys in api instance so that it's value (default value = false) can be modified by navitia-configurator.  
For details: https://navitia.atlassian.net/browse/NAV-1437

Relates to: https://github.com/hove-io/navitia-configurator/pull/699

### CODE REVIEW

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -992,6 +992,14 @@ def put(self, version=0, id=None, name=None):
             default=instance.ghost_words,
         )
 
+        parser.add_argument(
+            'filter_odt_journeys',
+            type=inputs.boolean,
+            help='boolean to activate / deactivate filter on odt journeys',
```
Comment (woshilapin) [inline source/tyr/tyr/resources.py]:
```suggestion
            help='boolean to activate / deactivate filter on on-demand transport journeys',
```

**Review by woshilapin — APPROVED** [review]

**Review by pbench — APPROVED** [review]

File: source/tyr/tyr/resources.py (hunk 1 above)
Comment (kadhikari) [reply source/tyr/tyr/resources.py]: :+1:

**Comment by SGrenet** [comment]
is it False by default?

**Review by SGrenet — APPROVED** [review]

**Review by woshilapin — APPROVED** [review]

**Comment by kadhikari** [comment]
> is it False by default?

Yes

**Pull request merged**